ui/src/search/config.js

```diff
@@ -79,7 +79,7 @@ const searchConfig = {
   },
   [EXPERIMENTS_NS]: {
     persistedQueryParamsDuringNewSearch: defaultPersistedQueryParamsDuringNewSearch,
-    onQueryChange: onAggregationlessCollectionQueryChange,
+    onQueryChange: onEventsQuerychange,
```

Comment:
maybe rename the `onEventsQuerychange` to something that fits to `seminars`, `conferences`, `experiments`

Reply:
Maybe `onCollectionQueryChange`?